components/TableInstance/TableInstance.jsx

```diff
@@ -13,10 +14,27 @@ class TableInstance extends Component {
     this.onClickRemove = this.onClickRemove.bind(this);
   }
 
-  componentDidMount() {
+  componentWillMount() {
     this.props.getInstanceList();
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (
+      nextProps.firstScreen &&
+      nextProps.instanceList.length === 0 &&
+      !nextProps.loading
+    ) {
+      hashHistory.push('/create-view');
+    }
+    if (nextProps.instanceList.length === 0 && !nextProps.loading) {
+      this.props.showNoInstancesFound();
```

Comment:
This changes the behavior of the current configurator. Remove this notification.